lib/ui/common/ActionSheet.dart

```diff
@@ -107,6 +103,12 @@ class ActionSheet extends StatefulWidget {
   final ActionSheetViewModel _viewModel;
   final ActionSheetStyle _style;
 
+  static present(ActionSheet sheet, BuildContext context) {
+    showModalBottomSheet(backgroundColor: Colors.transparent,
```

Comment:
Is not compiling with this param, I think there's no `backgroundColor` on `showModalBottomSheet` func

```suggestion
    showModalBottomSheet(
```

Reply:
![image](https://user-images.githubusercontent.com/23307893/61705116-7c397600-ad45-11e9-96e9-7ba74292cbdb.png)

Reply:
that's why the CI´s not working

I did update my flutter :(

Reply:
just remove this and it works :D

Reply:
:

Reply:
Need it to have the rounded corners however:(

Reply:
👍

Reply:
you'll have to do a flutter update for it to build. we need that line. just updated the CI to same version I'm using.

Reply:
👌  I'll update flutter then